### ISSUE TITLE
1700 stop

### ISSUE THREAD

**Issue by JX-Github-web** [robot0321/DepthRegularizedGS#24]
have you solved this problem yet?
![image](https://github.com/user-attachments/assets/72055e50-d71a-4336-ad78-56bbeeb6029c)

**Comment by stan1512** [comment]
train.py  135 
if iteration % 100 == 0:
                if iteration > opt.min_iters and ema_depthloss_for_log > prev_depthloss:
                    training_report(tb_writer, iteration, Ll1, loss, l1_loss, iter_start.elapsed_time(iter_end), [iteration], scene, render, 
                                    (pipe, background), txt_path = os.path.join(args.model_path, "metric.txt"))
                    scene.save(iteration)
                    print(f"!!! Stop Point: {iteration} !!!")
                    break
                else:
                    prev_depthloss = ema_depthloss_for_log
eliminate this early end logic

**Comment by JX-Github-web** [comment]
> 如果迭代 % 100 == 0，则 train.py 135：如果迭代 > opt.min_iters 且ema_depthloss_for_log > prev_depthloss：training_report（tb_writer， iteration， Ll1， loss， l1_loss， iter_start.elapsed_time（iter_end）， [iteration]， scene， render， （pipe， background）， txt_path = os.path.join（args.model_path， “metric.txt”）） scene.save（iteration） print（f“!!停止点： {iteration} !!”） break else：prev_depthloss = ema_depthloss_for_log 消除此早期结束逻辑

Thanks, this one works
